dev-docs/source/MVPTutorial/CompleteGUI.rst

```diff
@@ -54,8 +56,8 @@ which has the addition of the data and colour models being passed to
 the Presenter. This makes it easier for us to replace the Model at a
 later date.
 
-Master View
-###########
+``masterView.py``
```

Comment:
Could we rename these to follow PEP8 standards for module names (https://www.python.org/dev/peps/pep-0008/#package-and-module-names).